fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/InsertIntoTableCommand.java

```diff
@@ -136,15 +145,15 @@ public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
 
         if (isOverwrite) {
             dealOverwrite(ctx, executor, physicalOlapTableSink);
-            return;
+            return null;
```

Comment:
so stream load could not do insert overwrite?

Reply:
done